lib/rev-api/models/order_request.rb

```diff
@@ -101,24 +101,42 @@ def validate_inputs(inputs)
   # using a POST to /inputs, and are referenced using the URIs returned by that call. We also support external links.
   # @see https://www.rev.com/api/ordersposttranscription
   class TranscriptionOptions < InputOptions
+    # Optional, array of file formats the captions should be delivered as.
+    # default is MS Word
+    attr_reader :output_file_formats
+
     # Optional, should we transcribe the provided files verbatim? If true,
     # all filler words (i.e. umm, huh) will be included.
     attr_reader :verbatim
 
     # Optional, should we include timestamps?
     attr_reader :timestamps
 
+    # All supported output file formats
+    OUTPUT_FILE_FORMATS = {
+      :ms_word => 'MS Word',
+      :json => 'JSON',
+      :text => 'Text',
+      :pdf => 'Pdf'
```

Comment:
Do these work with Timi's recent changes?

Reply:
Yes, it looks like it https://github.com/revdotcom/revdotcom/pull/19036/

Timi's changes made parsing strings case-insensitive so case is fine, and the `output_file_formats` param is still the same name